packages/@okta/vuepress-theme-prose/assets/css/components/_card.scss

```diff
@@ -103,6 +103,10 @@
     }
   }
 
+  .card--header-img {
+    width: 4rem;
```

Comment:
May need to adjust this to work with horizontal logos (Yahoo Japan could be a test. I don't think they have an icon).